src/current/_includes/v23.1/sidebar-data/feature-overview.json

```diff
@@ -0,0 +1,82 @@
+{
+    "title": "Feature Overview",
+    "is_top_level": true,
+    "items": [{
+        "title": "CockroachDB Products",
+        "urls": [
+          "/${VERSION}/choose-a-deployment-option.html"
```

Comment:
I'd like the nav entry and the page title to match -- perhaps, "Deployment Options"